contributing/code/git.rst

```diff
@@ -40,3 +40,17 @@ adding ``--show-notes=github-comments`` to the ``git show`` command:
 .. code-block:: bash
 
     $ git show HEAD --show-notes=github-comments
+
+Branches
+--------
+
+Branches of lower versions are merged in branches of higher versions on a regular
+basis by the members of the core team.
+
+The ``--log`` option of the ``git merge`` command be used to include the list of
```

Comment:
A word is probably missing after `command`:

`command be used` -> `command can be used` ? `command must be used` ?